From 1b7b9037e20e33c8b1609ae8fdf7f1a1c6658731 Mon Sep 17 00:00:00 2001
From: Alexander Alemayhu <alexander@alemayhu.com>
Date: Tue, 18 Oct 2016 11:26:04 +0200
Subject: [PATCH 3/3] staging: rtl8188eu: remove redundant assignment

'pHalData->AntDivCfg = 0;' and 'pHalData->TRxAntDivType =
pHalData->TRxAntDivType;' does nothing.

Discovered using coccinelle and changed with following script:

@@
expression x;
@@

- x = x;

Signed-off-by: Alexander Alemayhu <alexander@alemayhu.com>
---
 drivers/staging/rtl8188eu/hal/rtl8188e_hal_init.c | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/drivers/staging/rtl8188eu/hal/rtl8188e_hal_init.c b/drivers/staging/rtl8188eu/hal/rtl8188e_hal_init.c
index edc6b528a7af..c34a4e35afc3 100644
--- a/drivers/staging/rtl8188eu/hal/rtl8188e_hal_init.c
+++ b/drivers/staging/rtl8188eu/hal/rtl8188e_hal_init.c
@@ -608,8 +608,7 @@ void Hal_ReadAntennaDiversity88E(struct adapter *pAdapter, u8 *PROMContent, bool
 		if (pHalData->TRxAntDivType == CG_TRX_HW_ANTDIV || pHalData->TRxAntDivType == CGCS_RX_HW_ANTDIV)
 			pHalData->AntDivCfg = 1; /*  0xC1[3] is ignored. */
 	} else {
-		pHalData->AntDivCfg = 0;
-		pHalData->TRxAntDivType = pHalData->TRxAntDivType; /*  The value in the driver setting of device manager. */
+		pHalData->AntDivCfg = 0;/*  The value in the driver setting of device manager. */
 	}
 	DBG_88E("EEPROM : AntDivCfg = %x, TRxAntDivType = %x\n", pHalData->AntDivCfg, pHalData->TRxAntDivType);
 }
-- 
2.10.1

